simulation/simtypes/legacy.go

```diff
@@ -58,6 +59,7 @@ func GenAndDeliverTx(
 ) (simulation.OperationMsg, []simulation.FutureOperation, error) {
 	account := ak.GetAccount(ctx, simAccount.Address)
 	tx, err := helpers.GenTx(
+		rand.New(rand.NewSource(time.Now().UnixNano())),
```

Comment:
Are you sure that we want `time.Now()` here? Do we want non-deterministic seed here?

Reply:
Agreed

Reply:
ah actually fixed in a PR revert